### PR TITLE
CVE-2015-1268 and CVE-2016-5132

### PR DESCRIPTION
**Pull request by noesterle**
I edited CVE-2015-1268 and CVE-2016-5132

Nathan Oesterle

### CODE REVIEW

**Comment by samshiffman321** [comment]
please make the title of your PR "CVE-2015-1268 and CVE-2016-5132"

File: cves/CVE-2015-1268.yml [hunk 1]
```diff
@@ -33,7 +35,7 @@ fixes_vcc_instructions: |
 fixes:
 - :commit: 542ab473e104b7df23e95f933bda41723794d4da
   :note: ''
-vccs: []
+vccs: [e637f3f0bc810ff2c69bbf109c4943a5af14c466, 882d19eaa47ce09c32471ae00be366cf0e38e97d, 65ff000a8119770a10733449d2adaa9ad514d91e]
```
Comment (xdmds) [inline cves/CVE-2015-1268.yml]: remove brackets

File: cves/CVE-2015-1268.yml [hunk 2]
```diff
@@ -16,10 +16,12 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description: 
-bounty:
-  date: 
-  amount: 
-  references: []
+  A Document Object Mapper could be returned with an incorrect scope, allowing the remote attaker to bypass the Same Origin Policy. This policy limits scripts from a webpage to only access information on a second webpage with the same protocal, domain, and host.
```
Comment (xdmds) [inline cves/CVE-2015-1268.yml]: do you mean Document Object Model wrapper?

File: cves/CVE-2015-1268.yml (hunk 1 above)
Comment (xdmds) [inline cves/CVE-2015-1268.yml]: no brackets

File: cves/CVE-2015-1268.yml [hunk 3]
```diff
@@ -172,4 +174,4 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: This vulnerability's VCCs were made over the course of 8 months, by two people, one of which will be the patcher of the vulnerability. This tells me that there could have been a design mistake that led to passing in a type that was not intended to be there, or was intended to be there and the consequences weren't understood. If it was not a design mistake, I'd suspec that it was a lack of knowledge about the affects of the type being passed in.
```
Comment (xdmds) [inline cves/CVE-2015-1268.yml]: suspect

File: cves/CVE-2016-5132.yml [hunk 4]
```diff
@@ -37,15 +37,13 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - :commit: dd5dd98f5f59f4f53274308ec0a7dca74aba6525
-  :note: ''
+  :note: 'Made June 21'
```
Comment (xdmds) [inline cves/CVE-2016-5132.yml]: use a date format standard instead?

File: cves/CVE-2016-5132.yml [hunk 5]
```diff
@@ -171,8 +169,8 @@ lessons:
     applies: 
     note: 
   complex_inputs:
-    applies: 
-    note: 
+    applies: true
+    note: This exploit involves a very complex set of inputs, inlvolving iframesfrom unsecure page opening a sercure page, allowing some data to be intercepted.
```
Comment (xdmds) [inline cves/CVE-2016-5132.yml]: secure

File: cves/CVE-2016-5132.yml [hunk 6]
```diff
@@ -185,4 +183,4 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: This appears to be a design mistake. This vulnerability takes advantage of a very specific scenario that doesn't seem like it was accounted for, and took a loarge overhaul of many files to fix.
```
Comment (xdmds) [inline cves/CVE-2016-5132.yml]: large (spelling).

It sounds more like it was an issue with testing since proper equivalence partitioning wasn't implemented to cover edge cases from your description:

"The vulnerability had unit tests covering parts of it, but didn't provide the coverage needed to find this problem. This vulnerability was submitted by a user. The fix to this included altering unit tests."
